satzung.md

```diff
@@ -53,18 +53,18 @@
 * Gegen den Beschluss des Vorstandes ist die Anrufung der Mitgliederversammlung zulässig. Bis zum Beschluss der Mitgliederversammlung ruht die Mitgliedschaft.
 
 #### § 6 MITGLIEDSBEITRÄGE
-* Mitglieder entrichten Beiträge nach Maßgabe der jeweils aktuellen Beitragsordnung. In Sonderfällen hat der Vorstand des Vereins das Recht eine Einzelfallregelung per einfachen Mehrheitsbeschluss festzulegen.
+* Mitglieder entrichten Beiträge nach Maßgabe der jeweils aktuellen Beitragsordnung. In Sonderfällen hat der Vorstand des Vereins das Recht eine Einzelfallregelung per einfachem Mehrheitsbeschluss festzulegen.
 
 #### § 7 FINANZIERUNG
-* Der Verein finanziert die Durchführung seiner Aufgaben durch Mitgliederbeiträge, Spenden, Umlagen und andere finanzielle Mittel, soweit sie nicht den gemeinnützigen Zwecken des Vereins widersprechen.
-* Nicht-künstlerische Projekte können auch für den Eigenbedarf realisiert werden, in diesem Falls allerdings gegen ein vom Verein festgelegtes Entgelt.
+* Der Verein finanziert die Durchführung seiner Aufgaben durch Mitgliedsbeiträge, Spenden, Fördermittel, Umlagen und andere finanzielle Mittel, soweit sie nicht den gemeinnützigen Zwecken des Vereins widersprechen.
 
 #### § 8 ORGANE
-* Die Organe der Welcome Werkstatt sind die Mitgliederversammlung und der Vorstand.
+* Die Organe der Welcome Werkstatt sind die Mitgliederversammlung, der Vorstand und der Beirat.
```

Comment:
Motivation: Der Beirat soll als Organ in die Satzung aufgenommen werden aus versicherungstechnischen Gründen. Als offizielles Vereinsorgan sind Beiratsmitglieder über unsere Vereins-Haftpflicht automatisch mit-versichert.